lib/hammer_cli_foreman/hostgroup.rb

```diff
@@ -73,19 +73,28 @@ class InfoCommand < HammerCLIForeman::InfoCommand
 
       output ListCommand.output_definition do
         field :description, _("Description"), Fields::LongText, :hide_blank => true
-        field nil, _("Subnet"), Fields::SingleReference, :key => :subnet
-
-        field nil, _("Domain"), Fields::SingleReference, :key => :domain
-        field nil, _("Architecture"), Fields::SingleReference, :key => :architecture
-        field nil, _("Partition Table"), Fields::SingleReference, :key => :ptable
-        field nil, _("Medium"), Fields::SingleReference, :key => :medium
+        field :ancestry, _("Parent Id"), Fields::Field, :hide_blank => true
+        field :parent_name, _("Parent Name"), Fields::Field, :hide_blank => true
```

Comment:
You can use
```ruby
field nil, _("Parent"), Fields::SingleReference, :key => :parent
```
here and remove the ancestry field. Parent id is returned also in `:parent_id`.